src/profile/components/basicData/__tests__/BasicData.test.tsx

```diff
@@ -1,22 +1,31 @@
 import React from 'react';
 import { act } from '@testing-library/react';
 
-import { getMyProfile } from '../../../../common/test/myProfileMocking';
+import {
+  cloneAndManipulateProfile,
+  getMyProfile,
+} from '../../../../common/test/myProfileMocking';
 import {
   renderProfileContextWrapper,
   TestTools,
   RenderChildrenWhenDataIsComplete,
+  cleanComponentMocks,
+  WaitForElementAndValueProps,
+  ElementSelector,
 } from '../../../../common/test/componentMocking';
 import { ProfileData } from '../../../../graphql/typings';
 import BasicData from '../BasicData';
 import {
   MockedResponse,
   ResponseProvider,
 } from '../../../../common/test/MockApolloClientProvider';
-import { BasicDataValue } from '../../../helpers/editData';
+import { basicDataType, BasicDataValue } from '../../../helpers/editData';
+import i18n from '../../../../common/test/testi18nInit';
 
 describe('<BasicData /> ', () => {
-  const renderTestSuite = (responses: MockedResponse[]) => {
+  const responses: MockedResponse[] = [];
+  const initialProfile = getMyProfile().myProfile as ProfileData;
+  const renderTestSuite = () => {
```

Comment:
I would probably move this closer to usage, to line 92, above _initTests_ function.

Reply:
Referring to another re-ordering comment:
https://github.com/City-of-Helsinki/open-city-profile-ui/pull/154#discussion_r618158453